### ISSUE TITLE
OpenType features do not work (Missing OTF in Google Fonts package)

### ISSUE THREAD

**Issue by keikhcheung** [floriankarsten/space-grotesk#42]
Take a look at the Space Grotesk [website](https://floriankarsten.github.io/space-grotesk/) itself and scroll down to the section demonstrating the OpenType features. 

It seems that this problem originates from [Google Fonts](https://fonts.google.com/specimen/Space+Grotesk). Only TTF fonts are included in the package hosted there.

**Comment by iskrisis** [comment]
Yeah it seems it's a Google Fonts issue. We can fix the presentation site by linking to the static version but this also probably means the stylistic sets are broken for anyone using Google Fonts.

**Comment by mightybart** [comment]
Temporarily fixed as described above, will check later if it's somehow possible to use OT features with GF loaded fonts.

**Comment by dahaca** [comment]
Will that Google Fonts issue be resolved? I would like to use the GFonts instead of static files but need to change the stylistic sets.

**Comment by mightybart** [comment]
I don't know, we can't fix it on our side. But this may be a good place to ask 👉 https://github.com/google/fonts/issues/1335

**Comment by dahaca** [comment]
> I don't know, we can't fix it on our side. But this may be a good place to ask 👉 [google/fonts#1335](https://github.com/google/fonts/issues/1335)

Oh, thought a wrong version was uploaded :D 

Thank you for a quick reply!

**Comment by milovangudelj** [comment]
Is this still an unresolved issue?

Loading the font from GF gives me the optimisation that I need but it doesn't have a lot of glyphs like `→`.

While loading it from a static file gives me all the glyphs that I need but at the cost of some layout shift caused by lack of loading optimisation.

To be more clear. I'm using `Next.js` to build my website and it does a great job at optimising web font loading, but that only works with fonts coming from GF and Typekit. Locally loaded fonts don't get the same treatment.
